appveyor.yml

```diff
@@ -30,11 +33,11 @@ environment:
           PYTHON_ARCH: "64"
           PYTHON_VERSION: "2.7"
 
-        - PYTHON_HOME: "C:\\Miniconda3"
+        - PYTHON_HOME: "C:\\Miniconda34"
           PYTHON_ARCH: "32"
           PYTHON_VERSION: "3.4"
 
-        - PYTHON_HOME: "C:\\Miniconda3-x64"
+        - PYTHON_HOME: "C:\\Miniconda34-x64"
```

Comment:
Side note: We can just use the latest Miniconda as the Python version used by `conda` doesn't affect the Python version we use in our testing environment.